src/Home/ShareLink.tsx

```diff
@@ -17,30 +24,31 @@ import {
   Affordances,
 } from "../styles"
 
-const ShareLink: FunctionComponent = () => {
+interface ShareLinkProps {
+  appDownloadUrl: string
+}
+
+const ShareLink: FunctionComponent<ShareLinkProps> = ({ appDownloadUrl }) => {
   const { applicationName } = useApplicationName()
   const { t } = useTranslation()
-  const configuration = useConfigurationContext()
-
-  const showShareLink = Boolean(configuration.appDownloadLink)
 
   const handleOnPressShare = async () => {
     try {
       await Share.share({
         message: t("home.bluetooth.share_message", {
           applicationName,
-          appDownloadLink: configuration.appDownloadLink,
+          appDownloadUrl,
         }),
       })
     } catch (error) {
-      throw new Error(error)
```

Comment:
Did not seem correct to me to crash the app if the link is not provided by accident.